src/Raven.Server/RavenServer.cs

```diff
@@ -439,6 +451,29 @@ private void StartMonitoringCpuCredits()
                         Logger.Operations("Unhandled exception occured during cpu credit monitoring", e);
                 }
 
+                try
+                {
+                    if (sw.Elapsed.TotalSeconds >= (int)Configuration.Server.CpuCreditsExecSyncInterval.AsTimeSpan.TotalSeconds)
```

Comment:
use TimeSpan comparison

Reply:
Why? I don't like the compare functions where you have an integer result. 
Using the >= operator does the exact same thing: https://referencesource.microsoft.com/#mscorlib/system/timespan.cs,164